homeassistant/components/alarm_control_panel/ifttt.py

```diff
@@ -0,0 +1,106 @@
+"""
+Interfaces with alarm control panels that have to be controlled through IFTTT.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/alarm_control_panel.ifttt_securitysystem/
```

Comment:
The url should end with the platform name after the period, in this case that's `ifttt`.